### PR TITLE
OGS simulation

### PR DESCRIPTION
**Pull request by tStehling**
Wrap for the OGS simulation.

### CODE REVIEW

File: tools/ogs/ogs_simulation.xml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        #end if     
+        
+        #for $file in $xml_patch_files:
+            #set $identifier= re.sub(r'[^\w\-.]', '_', $file.element_identifier)
+            --xml-patch '$identifier' 
+        #end for
+       -o output
+    ]]></command>
+    <inputs>
+        <param name="project_file" format="prj" type="data" label="Project File (.prj)"/>
```
Comment (bernt-matthias) [inline tools/ogs/ogs_simulation.xml:38]: remove the `(.prj)` from the label.

File: tools/ogs/ogs_simulation.xml [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    </tests>
+
+    <help><![CDATA[
+        **Overview:**
+
+            This tool runs an OpenGeoSys (OGS) simulation with support for Python boundary conditions and enhanced CLI argument parsing. The simulation uses a project file and optional XML patch files to define and adjust simulation parameters dynamically.
+
+        **Inputs:**
+
+            1. **Project File:** The main project file (.prj) that defines the simulation setup.
```
Comment (bernt-matthias) [inline tools/ogs/ogs_simulation.xml:105]: Project files are missing here.

File: tools/ogs/ogs_simulation.xml [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+        #for $file in $xml_patch_files:
+            #set $identifier= re.sub(r'[^\w\-.]', '_', $file.element_identifier)
+            --xml-patch '$identifier' 
+        #end for
+       -o output
+    ]]></command>
+    <inputs>
+        <param name="project_file" format="prj" type="data" label="Project File (.prj)"/>
+        <param name="project_files" format="data" type="data_collection" collection_type="list" label="Project Files"/>
+        <param name="xml_patch_files" format="xml" type="data_collection" collection_type="list" label="XML Patch Files"/>
```
Comment (bernt-matthias) [inline tools/ogs/ogs_simulation.xml:40]: We should leave some words (maybe in the tool help) on the correct naming of the files.

Comment (tStehling) [reply tools/ogs/ogs_simulation.xml:40]: Correct Naming?
